Makefile

```diff
@@ -28,10 +28,11 @@ CFLAGS	= -Wall -Wextra -Werror $(EXTRA_CFLAGS) -pthread -DVERSION=$(VERSION) \
 LDLIBS	= -lm -lrt -pthread -lnl-genl-3 -lnl-3 $(EXTRA_LDFLAGS)
 OBJS	= esmc_socket.o dpll_mon.o nl_dpll.o synce_clock.o synce_dev.o \
 	  synce_dev_ctrl.o  synce_msg.o synce_port.o synce_port_ctrl.o \
-	  synce_transport.o  synce_ext_src.o synce_clock_source.o config.o \
+	  synce_ql_manager.o synce_transport.o  synce_ext_src.o synce_clock_source.o config.o \
 	  hash.o interface.o print.o util.o
 HEADERS = $(OBJS:.o=.h)
 BINARY 	= synce4l
+QL_APP 	= ql_manager_app
```

Comment:
I think we should go with more generic approach/naming, right now this API allows only for change of external QL, and read of the QL, but we have to make it more user friendly and expandable, we could also find a name which will allow for this, looking on the PTP they have pmc which stands for PTP Management Client, as i checked smc (SyncE Management Client) doesn't look like a conflict for other Linux apps.. so maybe this one?

Reply:
changed it. 
Based on that also renamed synce_ql_manager -> synce_manager